### PR TITLE
Update Policy-Controller on GCP with KMS post

### PR DESCRIPTION
**Pull request by mathieu-benoit**
- Use Workload Identity to use security best practice (least privilege GSA/KSA) with GKE and Policy-controller.
- Simplify the use of products/projects names: Cosign and Policy-controller
- Update Cosign logo as per https://github.com/sigstore/community/issues/239
- Update Policy-controller as per https://github.com/sigstore/community/pull/257
- Fix a few typos

### CODE REVIEW

**Comment by mathieu-benoit** [comment]
Quick one for small improvements. Ready for your review, thanks!

File: content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md [hunk 1]
```diff
@@ -129,52 +129,67 @@ The following checks were performed on each of these signatures:
...
 
-Install the [Sigstore’s `policy-controller` Helm chart](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller) in this GKE cluster:
+Install the [Policy-controller Helm chart](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller) in this GKE cluster by annotating the Policy-controller's `ServiceAccounts` to use Workload Identity:
 ```bash
 helm repo add sigstore https://sigstore.github.io/helm-charts
 helm repo update
-helm install policy-controller \
-    -n cosign-system sigstore/policy-controller \
-    --create-namespace
+helm upgrade policy-controller \
```
Comment (vaikas) [inline content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:186]: Is this the same as 'apply', meaning you can upgrade from 'nothing' :)

**Review by vaikas — APPROVED** [review]

File: content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md (hunk 1 above)
Comment (mathieu-benoit) [reply content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:186]: Yep, good question, `helm upgrade --install` will do the same as `helm install` for the first time.

Comment (hectorj2f) [reply content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:186]: Yes, I'd prefer to use install here too.

**Review by hectorj2f — APPROVED** [review]
lgtm

**Comment by mathieu-benoit** [comment]
1 more reviewer/approver to go, thanks!

**Review by bobcallaway — APPROVED** [review]

**Comment by mathieu-benoit** [comment]
Thanks @bobcallaway!

Actually, @vaikas, could you approve it too please?

Thanks everyone!

**Review by vaikas — DISMISSED** [review]
Woot Woot! Thank you for doing this!

**Comment by mathieu-benoit** [comment]
Thanks everyone, but this needs another round of 3 approvers especially from @sigstore/sigstore-blog-maintainers, thanks again!

Note: I just updated the image with the new Policy-controller logo 🎉

**Review by vaikas — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Pull request merged**